src/quail/sensors/ADCTask.cpp

```diff
@@ -49,20 +85,29 @@ void ADCTask::activity()
             long ret = sys.adc.getData(adc_data.dataword, adc_data.channel);
             // turn interrupt back on to catch new ready indicator
             sys.adc.setIRQAction(adcISR);
+            //Serial.println("C");
+
             if( ret >= 0) {
                 // do thing with data
-                (sys.sensors[adc_data.channel])->addADCdata(adc_data.dataword);
+                if(i > 1000){
```

Comment:
should we drop this before pushing to master? Or at least change it to more clear message like "ADC healthy"

Reply:
nah its fine